crest/Assets/Crest/Crest/Scripts/LodData/RegisterClipSurfaceInput.cs

```diff
@@ -43,25 +41,23 @@ public enum Primitive
         bool _enabled = true;
         public override bool Enabled => _enabled;
 
-        [Header("Clip Surface Input Options")]
-
-        [Tooltip("Where the source of the clipping will come from.")]
-        [SerializeField]
-        internal Mode _mode = Mode.Primitive;
+        [Header("--- PRIMITIVE MODE ---")]
 
         [Tooltip("The primitive to render (signed distance) into the simulation.")]
         [SerializeField, Predicated("_mode", inverted: true, Mode.Primitive), DecoratedField]
         Primitive _primitive = Primitive.Cube;
 
+        // TODO is this really specific to PRIMITIVE mode?
```

Comment:
could you look at this and the below? think i was too enthusiastic with the predication

Reply:
ah cool added these notes and removed todos